src/base/bittorrent/torrenthandle.cpp

```diff
@@ -68,6 +68,20 @@ const QString QB_EXT {".!qB"};
 namespace libt = libtorrent;
 using namespace BitTorrent;
 
+namespace
+{
+    using EntryDict = std::map<std::string, libt::entry>;
+
+    EntryDict setToEntryDict(const QSet<QString> &input)
+    {
+        EntryDict output;
+        const libt::entry::integer_type dummy_value(0);
```

Comment:
Yeah, sorry for making you guys point this minor style guide stuff out. I have to use the other style all day at work, so it is a battle against my instincts.